.github/workflows/README.md

```diff
@@ -0,0 +1,14 @@
+# Run GitHub Actions on Your Machine
+We use github actions to check if the code is sound and in the format we expect before it can be merged to master.
+It is possible to run the actions locally with [act](https://github.com/nektos/act).
+To run the `check-code-quality` action use the following command:
+```
+act -j check-code-quality
```

Comment:
```suggestion
act -r -j check-code-quality
```
Let's use it with `-r`, which says `don't remove container(s) on successfully completed workflow(s) to maintain state between runs`. It saves 10 min for me since it doesn't download flutter every time 😜